htdocs/comm/propal/class/propal.class.php

```diff
@@ -841,6 +869,13 @@ public function addline($desc, $pu_ht, $qty, $txtva, $txlocaltax1 = 0.0, $txloca
 					}
 
 					$this->lines[] = $this->line;
+				} else {
```

Comment:
I don't understand the goal of this portion of added code ?
Once we have inserted line (code line 857), it it is not in a clone process, we must recalculate the position (updateRangOfLine).
If not, whe should have nothing to do. The line has been inserted by the line->insert with all similar value than the cloned object.